kod/util/questengine.kod

```diff
@@ -400,19 +400,19 @@ resources:
    stunteddwarf_assign_3 = "Ah, an ancient drink.  Actually, the recipe has been lost in time, although I hear "
          "that %NPC might know of it."
    stunteddwarf_trigger_4 = "myth"
-   stunteddwarf_assign_4 = "Stunted dwarf?  Ha!  That�s just a myth, and you tell %HIMHER_NPC I said that!"
+   stunteddwarf_assign_4 = "Stunted dwarf?  Ha!  That's just a myth, and you tell %HIMHER_NPC I said that!"
    stunteddwarf_success_4 = "Ah, well.  If you ever hear anyone mention a stunted dwarf, you be sure to find out "
          "what you can."
    stunteddwarf_assign_6 = "Ah, you need a recipe for your garden variety stunted dwarf, eh?  Tell you what.  "
-         "Prove to me you�re hero enough to drink one, and I�ll give you the recipe.  Go kill a kriipa for me."
+         "Prove to me you're hero enough to drink one, and I�ll give you the recipe.  Go kill a kriipa for me."
```

Comment:
Can you double-check this one?  The replacement looks sus.

Reply:
Yes, a miss! Fixed.